fw/http_frame.c

```diff
@@ -2548,6 +2561,8 @@ do {									\
 			stream->xmit.is_blocked = !stream->rem_wnd;
 			T_FSM_EXIT();
 		}
+
+		ADJUST_TMP_AVAILABLE_CWND(*cwnd_awail, tmp_cwnd_awail);
```

Comment:
I can suggest do this on transition to next state from `HTTP2_MAKE_HEADERS_FRAMES` and `HTTP2_MAKE_CONTINUATION_FRAMES`, because check it every DATA frame which will be much more than HEADERS, not looks right.

Reply:
fixed